charts/cluster-addons/templates/openstack/csi-cinder.yaml

```diff
@@ -72,27 +72,55 @@ spec:
   releaseName: csi-cinder-storageclass
   manifestSources:
     - template: |
-        {{- with .Values.openstack.csiCinder.storageClass }}
+        {{- with .Values.openstack.csiCinder.defaultStorageClass }}
         apiVersion: storage.k8s.io/v1
         kind: StorageClass
         metadata:
           name: {{ .name }}
-          {{- if .isDefault }}
           annotations:
             storageclass.kubernetes.io/is-default-class: "true"
-          {{- end }}
         provisioner: cinder.csi.openstack.org
         parameters:
           availability: {{ .availabilityZone }}
           {{- with .volumeType }}
           type: {{ . }}
           {{- end }}
+          {{- with .fstype }}
+          fstype: {{ . }}
+          {{- end }}
         reclaimPolicy: {{ .reclaimPolicy }}
         allowVolumeExpansion: {{ .allowVolumeExpansion }}
-        volumeBindingMode: WaitForFirstConsumer
+        volumeBindingMode: {{ .volumeBindingModea }}
         {{- with .allowedTopologies }}
         allowedTopologies: {{ toYaml . | nindent 6 }}
         {{- end }}
         {{- end }}
+{{- $storeageClassIndex := 0 }}
```

Comment:
Typo

Also, what is this actually used for? As far as I can see it is incremented but never used.